source/codegen/metadata/niscope/config.py

```diff
@@ -1,12 +1,15 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-SCOPE API metadata version 23.0.0f168
+# This file is generated from NI-SCOPE API metadata version 23.3.0d9999
 config = {
     'additional_headers': {
         'custom/ivi_errors.h': [
             'service.cpp'
-        ]
+        ],
+        'niScopePrivate.h': [
```

Comment:
Note: This is how I got it to pull in the private header. I didn't see a way to pull in multiple headers currently with the `c_header` key so just did it this way.